src/main/java/reactor/rabbitmq/ChannelCloseHandlers.java

```diff
@@ -0,0 +1,30 @@
+package reactor.rabbitmq;
+
+import com.rabbitmq.client.Channel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import reactor.core.publisher.SignalType;
+
+import java.util.function.BiConsumer;
+
+public class ChannelCloseHandlers {
+
+    public static class SenderChannelCloseHandler implements BiConsumer<SignalType, Channel> {
+
+        private static final Logger LOGGER = LoggerFactory.getLogger(SenderChannelCloseHandler.class);
+
+        @Override
+        public void accept(SignalType signalType, Channel channel) {
+            int channelNumber = channel.getChannelNumber();
+            LOGGER.info("closing channel {} by signal {}", channelNumber, signalType);
```

Comment:
Are you sure you want to log every channel closure as `info`? Some environments have high channel churn.

Reply:
I would prefer to log it as `debug`. It is extracted from here 
[channel close](https://github.com/reactor/reactor-rabbitmq/blob/master/src/main/java/reactor/rabbitmq/Sender.java#L123-L130 )

Reply:
I changed logging level to `debug`